scripts/benchmark_ipa.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Generate stage-wise report after running IPA bench
+#
+# To use, supply the number of rows you want to run benchmarking on
+
+num_rows=$1
+if [ "$#" -ne 1 ]; then
+    echo "Please supply query size as an argument to the script"
+    exit
+fi
+
+temp_file=".benchmark_result$num_rows.csv"
```

Comment:
```suggestion
temp_file=`mktemp -q /tmp/ipa-bench.XXXXXX`
 if [ $? -ne 0 ]; then
         echo "$0: Can't create temp file, exiting..."
         exit 1
 fi
```

that way, even if someone presses ctrl-c while running ipa bench, they won't face "file exists" error message after re-running this script

Reply:
Didnt know about mktemp.. it looks nice!